install_config/web_console_customization.adoc

```diff
@@ -376,6 +376,41 @@ assetConfig:
 ----
 ====
 
+. As a cluster administrator, run the following command to give users the
```

Comment:
yeah, i'm pretty sure that guy just didn't run reconcile roles which is a standard part of the upgrade process.